### PR TITLE
global: docs and installation fixes

### PR DESCRIPTION
**Pull request by alizeepace**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -28,7 +28,7 @@ User's Guide
 ------------
 
 This part of the documentation will show you how to get started in using
-Invenio-Base.
+Invenio-Userprofiles.
```
Comment (tiborsimko) [inline docs/index.rst]: The canonical name of the package is `Invenio-UserProfiles`.

File: setup.py [hunk 2]
```diff
@@ -182,10 +151,9 @@ def run_tests(self):
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
```
Comment (tiborsimko) [inline setup.py]: Please remove 3.3 and 3.4 also from `.travis.yml`.

File: setup.py [hunk 3]
```diff
@@ -182,10 +151,9 @@ def run_tests(self):
...
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: Implementation :: CPython',
+        'Programming Language :: Python :: Implementation :: PyPy',
```
Comment (tiborsimko) [inline setup.py]: Please remove the PyPy line, as we don't test for pypy during CI yet.

**Comment by jirikuncar** [comment]
@alizeepace I have removed a commit that was adding Flask>=0.11.1 second time to setup.py. Please do not force push to this branch. Thanks!

**Pull request merged**